### PR TITLE
1378 old api removal

### PR DESCRIPTION
**Pull request by hwindsor**
## Pull Request Overview

- Removal of the old site settings API which is now unused now that https://github.com/WildMeOrg/codex-frontend/pull/449 is merged in

---

### CODE REVIEW

**Comment by Emily-Ke** [comment]
@hwindsor https://github.com/WildMeOrg/codex-frontend/pull/454 has been merged.

**Review by drewblount — APPROVED** [review]
LGTM

**Review by Atticus29 — COMMENTED** [review]
To the extent that I can understand this, it makes sense to me.

File: app/modules/site_settings/models.py [hunk 1]
```diff
@@ -609,13 +594,6 @@ def set_rest_block_data(cls, data):
                     cls.set_key_value(key, data[key])
                 else:
                     cls.forget_key_value(key)
```
[the remark this answers is not in the document]
Comment (hwindsor) [reply app/modules/site_settings/models.py:596]: No this was a bit of a hangover from EDM & houston both having some data. The function used to set the houston data and pass back anything it had not done to be sent onto EDM.
This is now totally unnecessary

**Review by naknomum — APPROVED** [review]
looks good to me.

**Pull request merged**